### DISCUSSION TITLE
#38 togETHer biweekly meeting

### DISCUSSION THREAD

**Discussion by Riely007** [tog-ETH-er/togETHer#14] [General]
Agenda:

1. Community updates
2. [ETH Shanghai](https://twitter.com/ETHShanghai2022) update
3. Fireside discussion

> 会议主题：togETHer
> 会议时间： 21:00-22:00
> 重复周期：每两周 (周一)
> 
> 点击链接入会，或添加至会议列表：
> https://meeting.tencent.com/s/LxNajKWJw4RV
> 
> 会议 ID：891 5098 6775

**Comment by Riely007** [comment]
### 以太坊翻译小组

1. 完成以太坊近期[报告的翻译](https://docs.google.com/document/d/1OaRNE8L7Og7vXlTHOKfWrpYiNL18HdbkjzWwAS8Xijw/edit?disco=AAAAYL54wQM&usp_dm=true)
2. 完成短袖周边的设计，联系供应商制作中

**Comment by Thegaram** [comment]
### Crypto Tech Night

04.28 举办了CTN第13期，活动主题为以太坊合并，感谢 @nake13 @winkybupt @bambalados. 这次一共有～80个人参加。回放：https://www.bilibili.com/video/BV16S4y1a719

5月我们准备举办一次Twitter Spaces上的“闲聊”，还有CTN #14，主题暂定 layer 2 (Arbitrum, Scroll, etc)

**Comment by 0x-stan** [comment]
# Dapp-Learning

近两周工作进展

## repo update

33个PR #703 ～ #736

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pulls

## basic task translate

开始针对前29个基础任务的README进行中转英翻译，已经召集6位小伙伴分工合作

## sharing meeting

近两周举办了3场分享会

- 《Euler白皮书解析》by Stacy 
  - https://youtu.be/1pyHuE2qMLE
  - https://youtu.be/mDeoYknMx_M

-《CurveDAO源码解读》 by ququzone
  - https://www.youtube.com/watch?v=aDyTEHP8Ns0
  - https://www.youtube.com/watch?v=eiSnHhlwqBA

-《Sushi源码解读》by Joly
  - https://www.youtube.com/watch?v=Lm0ympf-rVA
  - https://www.youtube.com/watch?v=NLvsx6kbUfE

## 翻译小组

- 审稿阶段第 14/15集

## Defi 代码详解系列文章开坑

《Uniswap v1 代码详解》：<https://mp.weixin.qq.com/s/Qqa4SmbZMpxP6Y2hfFf4sQ>

后续会按照 Uniswap v1-v2-v3 Curve-v1-v2 的defi学习路线图更新

**Comment by winkybupt** [comment]
# 原语里弄
1. 参加 Crypto tech night, 讨论 The Merge 相关话题
2. 继续共读活动，并举办互联网之子放映活动

**Comment by bobjiang** [comment]
- GitcoinDAO S14 budget request
- Gitcoin China proposal

**Comment by stephanieeee-ho** [comment]
### ECN
1. 常规的翻译与周报制作
2. The Merge 主题活动的筹备，目前邀约了 Hsiao-Wei、Lido、Rocket Pool 做分享和 AMA，诚邀社区支持

**Comment by 0x68616f79756e** [comment]
ETHPlanet
1.协助ETHShanghai
2.全力筹备EDCON以及ETAC活动

**Comment by Emilyxiannv** [comment]
### DAOSquare


DAOSquare在社区分享了一个Web3的内容创作者协议的项目：MuseX，它是一个关注Ownship与Shareship的去中心化开放创作协议，虽然MuseX很关注价值的创造, 但也不会忽视创作体验, 所以MuseX还提供了基于NFT驱动的链上权限管理与完善的编辑器支持.

在节后会上线，有兴趣的可以体验，相关链接：https://www.musex.io/

另外最近翻译了一些Web3和DAO的一些文章
2022年的所有权经济： https://mp.weixin.qq.com/s/k3MQkwNv_O6qraD7Y6eiig
我眼中的创作者经济：https://mp.weixin.qq.com/s/_m5q9IZ4E506oqVuVr9KiQ
